app/uk/gov/hmrc/trusts/utils/DomainValidator.scala

```diff
@@ -179,7 +197,9 @@ object BusinessValidation {
       domainValidator.trustStartDateIsNotFutureDate,
       domainValidator.validateEfrbsDate,
       domainValidator.trustEfrbsDateIsNotFutureDate,
-      domainValidator.deceasedSettlorDobIsNotFutureDate
+      domainValidator.deceasedSettlorDobIsNotFutureDate,
+      domainValidator.deceasedSettlorDoDIsNotFutureDate,
```

Comment:
Duplicate ?

Reply:
@jassalrichy  it is not duplicate, it is different, it is for date of death